src/EdFi.AnalyticsMiddleTier.DataStandard32/Engage/MSSQL/0000-Table-GradeBands-Create.sql

```diff
@@ -0,0 +1,23 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE analytics.engage_GradeBands (
```

Comment:
I just realized that this should have been `analytics_config` instead of `analytics`. I'll create a separate ticket so that we can deal with it later.

Reply:
No, correcting it now.